src/prommis/precipitate/precipitate_liquid_properties.py

```diff
@@ -76,30 +76,89 @@ def build(self):
         self.Gd = Component()
         self.Dy = Component()
 
-        # parameter based on pH 1.28
+        # parameter based on pH 1.5
         # TODO add surrogate model/equation
-        self.split = Param(
+        self.split = Var(
```

Comment:
I'm assuming these values are divided by 100 so that you don't have to explicitly do so in the precipitator unit model file? If so, should the upper bound be 1 instead of 100?